src/download-source.html

```diff
@@ -54,9 +43,6 @@ <h2 class="narrow">Information</h2>
   <p>
     See also <a href="https://wiki.sagemath.org/ReleaseTours/sage-{{version_src}}">Release Tour for version {{version_src}}</a> and the <a href="changelogs/sage-{{version_src}}.txt">changelog</a>.<br />
   <p>
-  You can <a href=
-  "https://git.sagemath.org/">browse all the
-  tracked source code repositories</a> and see exactly what&#39;s going on, and who did what
-  when.<br />
+  You can <a href="https://github.com/sagemath/sage/">browse the Sage repository on GitHub</a>.<br />
```

Comment:
Say something about other Sage parts in https://github.com/sagemath/

Reply:
well, do you want to add something as part of this PR? as you wish. you can make another one later as well.

Reply:
done